### ISSUE TITLE
validate 3.3.0 snapshot produces incorrect SKIP/INFO messages

### ISSUE THREAD

**Issue by rchenatjpl** [NASA-PDS-Incubator/validate#652]
### Checked for duplicates

No - I haven't checked

### 🐛 Describe the bug

The 5 SKIPs in the output are wrong. If those messages didn't exist, validate would be doing the right thing, including actually validating the files the SKIPs are complaining about.

```
% validate -V
gov.nasa.pds:validate
Version 3.3.0-SNAPSHOT
Release Date: 2023-05-31 18:10:35
Copyright 2019, by the California Institute of Technology ("Caltech").
All rights reserved.
% 
% 
% 
% validate -R pds4.bundle -t bug
PDS Validate Tool Report
Configuration:
   Version                       3.3.0-SNAPSHOT
   Date                          2023-06-01T07:22:05Z
Parameters:
   Targets                       [file:/Users/rchen/Desktop/bug/]
   Rule Type                     pds4.bundle
   Severity Level                WARNING
   Recurse Directories           true
   File Filters Used             [*.xml, *.XML]
   Data Content Validation       on
   Product Level Validation      on
   Allow Unlabeled Files         false
   Max Errors                    100000
   Registered Contexts File      /Users/rchen/PDS4tools/validate/resources/registered_context_products.json
Product Level Validation Results
  SKIP: file:/Users/rchen/Desktop/bug/document/archivedescription.xml
      INFO  [info.integrity.unreferenced_file]   Skipping file:/Users/rchen/Desktop/bug/document/archivedescription.xml due to collection not latest or does not sharing the same logical_identifier as the bundle target
  SKIP: file:/Users/rchen/Desktop/bug/document/ref.xml
      INFO  [info.integrity.unreferenced_file]   Skipping file:/Users/rchen/Desktop/bug/document/ref.xml due to collection not latest or does not sharing the same logical_identifier as the bundle target
  SKIP: file:/Users/rchen/Desktop/bug/document/lmi_shutterdelay_levine18.xml
      INFO  [info.integrity.unreferenced_file]   Skipping file:/Users/rchen/Desktop/bug/document/lmi_shutterdelay_levine18.xml due to collection not latest or does not sharing the same logical_identifier as the bundle target
  SKIP: file:/Users/rchen/Desktop/bug/document/AA_Document_Index.xml
      INFO  [info.integrity.unreferenced_file]   Skipping file:/Users/rchen/Desktop/bug/document/AA_Document_Index.xml due to collection not latest or does not sharing the same logical_identifier as the bundle target
  SKIP: file:/Users/rchen/Desktop/bug/document/bundle_description.xml
      INFO  [info.integrity.unreferenced_file]   Skipping file:/Users/rchen/Desktop/bug/document/bundle_description.xml due to collection not latest or does not sharing the same logical_identifier as the bundle target
  PASS: file:/Users/rchen/Desktop/bug/bundle_gbo.ast-near-sun.images.xml
        1 product validation(s) completed
  PASS: file:/Users/rchen/Desktop/bug/document/archivedescription.xml
        2 product validation(s) completed
  PASS: file:/Users/rchen/Desktop/bug/document/ref.xml
        3 product validation(s) completed
  PASS: file:/Users/rchen/Desktop/bug/document/lmi_shutterdelay_levine18.xml
        4 product validation(s) completed
  PASS: file:/Users/rchen/Desktop/bug/document/collection_gbo.ast-near-sun.images_document.xml
        5 product validation(s) completed
  PASS: file:/Users/rchen/Desktop/bug/document/AA_Document_Index.xml
        6 product validation(s) completed
  PASS: file:/Users/rchen/Desktop/bug/document/bundle_description.xml
        7 product validation(s) completed
PDS4 Bundle Level Validation Results
  PASS: file:/Users/rchen/Desktop/bug/document/collection_gbo.ast-near-sun.images_document.xml
        1 integrity check(s) completed
  PASS: file:/Users/rchen/Desktop/bug/bundle_gbo.ast-near-sun.images.xml
        2 integrity check(s) completed
  PASS: file:/Users/rchen/Desktop/bug/document/archivedescription.xml
        3 integrity check(s) completed
  PASS: file:/Users/rchen/Desktop/bug/document/lmi_shutterdelay_levine18.xml
        4 integrity check(s) completed
  PASS: file:/Users/rchen/Desktop/bug/document/AA_Document_Index.xml
        5 integrity check(s) completed
  PASS: file:/Users/rchen/Desktop/bug/document/bundle_description.xml
        6 integrity check(s) completed
  PASS: file:/Users/rchen/Desktop/bug/document/ref.xml
        7 integrity check(s) completed
Summary:
  0 error(s)
  0 warning(s)
  Product Validation Summary:
    7          product(s) passed
    0          product(s) failed
    5          product(s) skipped
  Referential Integrity Check Summary:
    7          check(s) passed
    0          check(s) failed
    0          check(s) skipped
End of Report
Completed execution in 3971 ms
```
[bug.zip](https://github.com/NASA-PDS/validate/files/11622330/bug.zip)

### 🕵️ Expected behavior

I expected [...]

### 📜 To Reproduce

1.
2.
3.
...

### 🖥 Environment Info

- Version of this software [e.g. vX.Y.Z]
- Operating System: [e.g. MacOSX with Docker Desktop vX.Y]
...

### 🦄 Related requirements

🦄 #xyz
